Project.toml

```diff
@@ -8,6 +8,7 @@ AMDGPU = "21141c5a-9bdb-4563-92ae-f87d6854732e"
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 CellArrays = "d35fcfd7-7af4-4c67-b1aa-d78070614af4"
+GLMakie = "e9467ef8-e4e7-5192-8a1a-b1aee30e663a"
```

Comment:
It's not optimal to have visualisation package and especially GLMakie as deps in the core project

Reply:
Thanks! 

I am still not entirely familiar on how to work with the *.toml files. I guess I simply required it for visualization in the script. How can I remove/avoid this? Simply just add "using GLMakie" at the beginning of my *.jl script?

Reply:
Sure. One option would be to have GLMakie in your base env. Then `using GLMakie` in other projects should "just" work. Else, you could create a tmp folder and project where you add both JustRelay and GLMakie. Last, you could add GLMakie locally to your JustRelax env, but not commit nor push the deps.

Reply:
For completion, you can remove a pkg from the `Project.toml` by typing `] rm GLMakie` for example

Reply:
Thanks! I already removed the package and included it in my base environment.